antarest/study/storage/utils.py

```diff
@@ -390,9 +390,14 @@ def export_study_flat(
 
     stop_time = time.time()
     duration = "{:.3f}".format(stop_time - start_time)
-    logger.info(f"Study {path_study} exported (flat mode) in {duration}s")
+    logger.info(
+        {
+            False: f"Study '{study_dir}' exported (flat mode) in {duration}s",
+            True: f"Study '{study_dir}' exported with outputs (flat mode) in {duration}s",
+        }[bool(outputs)]
```

Comment:
Do we really need to cast ?
Why not ternary syntax instead ? :) " ... if outputs else ..."

Reply:
Better: add "with outputs" or "without outputs" according to the boolean value